modules/tinyexr/image_saver_tinyexr.cpp

```diff
@@ -0,0 +1,179 @@
+#include "image_saver_tinyexr.h"
+
+#include "thirdparty/tinyexr/tinyexr.h"
+
+static bool is_supported_format(Image::Format p_format) {
+	// This is checked before anything else.
+	// Mostly 16-bit and 32-bit formats supported by Godot are considered.
+	// For others, use PNG.
+	switch (p_format) {
+		case Image::FORMAT_RF:
+		case Image::FORMAT_RGF:
+		case Image::FORMAT_RGBF:
+		case Image::FORMAT_RGBAF:
+		case Image::FORMAT_RH:
+		case Image::FORMAT_RGH:
+		case Image::FORMAT_RGBH:
+		case Image::FORMAT_RGBAH:
+			return true;
+		default:
+			return false;
+	}
+}
+
+static int get_pixel_type(Image::Format p_format) {
+	switch (p_format) {
```

Comment:
Do we want to support standard rgba8?

Reply:
I was wondering about this. The last type Tinyexr supports is `TINYEXR_PIXEL_TYPE_UINT`, but where is it specified that it would be 8-bit?

Reply:
I think openexr only supports 16bit and 32 bit floats, but you can save a regular 8bit int as a value from 0 to 1 directly.

Reply:
I guess I could use 16-bit float for these

Reply:
I would prefer to use the default 16bit openexr format for int8 colors.